custom/icds_reports/tasks.py

```diff
@@ -1684,7 +1684,7 @@ def _get_primary_data_for_cases(db, domain, day, case_type):
 
 @periodic_task_on_envs(
     settings.ICDS_ENVS,
-    run_every=crontab(minute=30, hour=12),  # To run on 6PM IST
+    run_every=crontab(minute=30, hour=0),  # To run on 6AM IST
```

Comment:
@calellowitz please tell if you feel running it on 6 AM can be any problem(in terms of aggregation). 
Basically want it to run after 12:00 AM night